daily-patch-information/src/main/java/org/wso2/patchinformation/constants/EmailConstants.java

```diff
@@ -0,0 +1,258 @@
+//
+// Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+package org.wso2.patchinformation.constants;
+
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.gmail.GmailScopes;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Email constant values
+ */
+public final class EmailConstants {
```

Comment:
This class could go inside the Constants class as an inner class. And in that instance please rename this as just 'Email'.